README.md

```diff
@@ -264,6 +267,37 @@ Each organization/tenant is associated to a different database.
 
 [Top](#top)
 
+###<a name="section3.4"></a>OrionMongoSink persistence
+Then `OrionMongoSink` will persist the data within the body as:
+
+    $ mongo
+    MongoDB shell version: 2.6.9
+    connecting to: test
+    > show databases
+    admin              (empty)
+    local              0.031GB
+    vehicles           0.031GB
+    test               0.031GB
+    > use vehicles
+    switched to db vehicles
+    > show collections
+    4wheels
+    4wheels_car1_car
+    4wheels_car1_car_speed
+    system.indexes
+    > db.4wheels.fnd()
+    { "_id" : ObjectId("5534d143fa701f0be751db82"), "recvTime" : "2015-04-20T12:13:22.41", "entityId" : "car1", "entityType" : "car", "attrName" : "speed", "attrType" : "kmh", "attrValue" : "112.9" }
+    > db.4wheels_car1_car.find()
+    { "_id" : ObjectId("5534d143fa701f0be751db82"), "recvTime" : "2015-04-20T12:13:22.41", "attrName" : "speed", "attrType" : "kmh", "attrValue" : "112.9" }
+    > db.4wheels_car1_car_speed.find()
+    { "_id" : ObjectId("5534d143fa701f0be751db82"), "recvTime" : "2015-04-20T12:13:22.41", "attrType" : "kmh", "attrValue" : "112.9" }
```

Comment:
The reason Cygnus "traditionally" stores the human-readable version of the timestamp and the timepstamp itself is because HDFS and CKAN have no conversion fucntions as MySQL or Mongo may have. Then, for homogeneity, all the sinks included both formats, despiete MySQL, as said, could offer such a functionality.